masterlist.yaml

```
@@ -25514,3 +25514,32 @@ plugins:
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
         udr: 3
+        
+  - name: 'SurWR.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/37982/'
+        name: 'Capital Whiterun Expansion'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
+        name: 'Rob''s Bug Fixes - Capital Whiterun Expansion'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - Capital Whiterun Expansion](https://www.nexusmods.com/skyrimspecialedition/mods/63355/)' ]
+        condition: 'checksum("SurWR.esp", D96CF5BF)'
+      - <<: *patchIncluded
+        subs: [ 'Creation Club - Fishing' ]
+        condition: 'active("ccBGSSSE001-Fish.esm") and not active("SurWR - Fishing.esp")'
+    clean:
+      - crc: 0xD96CF5BF
+        util: 'SSEEdit v4.0.4b'
+      - crc: 0x538498A4
+        util: 'SSEEdit v4.0.4b'
+
+  - name: 'SurWR - Fishing.esp'
+    url:
+      - link: https://www.nexusmods.com/skyrimspecialedition/mods/63355/
```

The link needs to be single-quoted:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```